UICatalog/Scenarios/ProgressBarStyles.cs

```diff
@@ -2,76 +2,74 @@
 using System.Linq;
 using System.Threading;
 using Terminal.Gui;
+using static UICatalog.Scenarios.Frames;
 
 namespace UICatalog.Scenarios {
 	[ScenarioMetadata (Name: "ProgressBar Styles", Description: "Shows the ProgressBar Styles.")]
 	[ScenarioCategory ("Controls")]
 	[ScenarioCategory ("Progress")]
 	[ScenarioCategory ("Threading")]
 
+	// TODO: Add enable/disable to show that that is working
+	// TODO: Clean up how FramesEditor works 
+	// TODO: Better align rpPBFormat
+
 	public class ProgressBarStyles : Scenario {
 		private Timer _fractionTimer;
 		private Timer _pulseTimer;
-		private const uint _timerTick = 100;
+		private const uint _timerTick = 20;
 
-		public override void Setup ()
+		public override void Init ()
 		{
+			Application.Init ();
+			ConfigurationManager.Themes.Theme = Theme;
+			ConfigurationManager.Apply ();
+
+			var editor = new FramesEditor () {
+				Title = $"{Application.QuitKey} to Quit - Scenario: {GetName ()}",
+				BorderStyle = LineStyle.Single
+			};
+			editor.ColorScheme = Colors.ColorSchemes [TopLevelColorScheme];
+
 			const float fractionStep = 0.01F;
-			const int pbWidth = 20;
+			const int pbWidth = 25;
 
 			var pbFormatEnum = Enum.GetValues (typeof (ProgressBarFormat)).Cast<ProgressBarFormat> ().ToList ();
 
 			var rbPBFormat = new RadioGroup (pbFormatEnum.Select (e => e.ToString ()).ToArray ()) {
 				X = Pos.Center (),
-				Y = 1
+				Y = 10
 			};
-			Win.Add (rbPBFormat);
+			editor.Add (rbPBFormat);
 
-			var ckbBidirectional = new CheckBox ("BidirectionalMarquee", true) {
+			var button = new Button ("Start timer") {
 				X = Pos.Center (),
 				Y = Pos.Bottom (rbPBFormat) + 1
 			};
-			Win.Add (ckbBidirectional);
-
-			var leftContainer = new View () {
-				Y = Pos.Bottom (ckbBidirectional) + 1,
-				Width = Dim.Percent (50),
-				Height = Dim.Fill ()
-			};
```

Comment:
You need to add these containers, otherwise the framed progress bar will not be visible.

Reply:
Not sure what you're talking about. Can you merge this to your PR so I can study?

Reply:
This @tig. Even with the terminal maximized you can't see all the progress bars on the screen. That why I had split it in two containers side by side.

![image](https://github.com/BDisp/Terminal.Gui/assets/13117724/93a388da-ead6-44a3-8144-20a2890dff2e)

Reply:
> This @tig. Even with the terminal maximized you can't see all the progress bars on the screen. That why I had split it in two containers side by side.
> 
> ![image](https://user-images.githubusercontent.com/13117724/283275268-93a388da-ead6-44a3-8144-20a2890dff2e.png)

This code was not in your PR when I pulled it down. My rewrite does not use left/right.

Reply:
I know that @tig and that why I think it should have. A smaller than my PR, otherwise you can't see the framed progress bars.

Reply:
I screwed up and somehow didn't submit all my changes before.

Look again and you'll see that how you're thinking about "framed progress bars" is different than how I'm seeing it.

Reply:
![image](https://github.com/BDisp/Terminal.Gui/assets/585482/bfed3220-0288-4c8c-98c0-ce757022b1fa)